l3build/l3build.lua

```diff
@@ -552,6 +552,28 @@ function filelist(path, glob)
   return files
 end
 
+-- Does what filelist does, but can also glob subdirectories. In the returned
+-- table, the keys are paths relative to the given starting path, the values
+-- are their counterparts relative to the current working directory.
+function tree(path, glob)
+  function cropdots(path)
```

Comment:
I did not know enough Lua to suspect that this might be a global declaration.

    local cropdots = function(path)

Is this what you want to indicate? Or should `tree` be local?